modules/tinymce/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - New `text_patterns_lookup` option to provide additional text patterns dynamically #TINY-8778
+- New SPAP element to editor to promote premium #TINY-8840
```

Comment:
We need to document the `promotion` option to be able to disable this... additionally no customer is going to know what SPAP means. So maybe something like this:
```suggestion
- New promotion element has been added to the UI and can be disabled using the new `promotion` option #TINY-8840
```

Reply:
I wasn't sure if we were to document the option or not, due to some suggestions that it shouldn't be. But I might have misunderstood something.